### ISSUE TITLE
Retention, pathways, selection

### ISSUE THREAD

**Issue by CaptainSifff** [the-teachingRSE-project/institutionalized_education#6]
We have the discussion on retention carried over from the-teachingRSE-project/competencies#78 , the-teachingRSE-project/competencies#84
It raises the issue of retaining the good and qualified people @tobyhodges wanted to contribute sth. here that highlights how retaining experienced people is better than constantly replacing them by young and cheap persons.
We can work in the notion that we take constant skill-up, which in academia has traditionally been done by the researchers themselves, to a new level, by doing this for our entire domain(See section on structural changes.)
Please note, that this also touches the issues of career pathways for RSEs.

**Comment by mschwarzmeier** [comment]
how is this related to the-teachingRSE-project/competencies#104 ?

**Comment by jcohen02** [comment]
As suggested in the earlier version of the above message, I think this may be somewhat duplicating the content in the-teachingRSE-project/competencies#104, @mschwarzmeier.

@CaptainSifff, might it make sense to merge the two?

**Comment by CaptainSifff** [comment]
Yes, It totally does.... Happens when you do not do this type of work in a continuous manner.

**Comment by CaptainSifff** [comment]
Copying over the description from the previous discussion.

**Comment by CaptainSifff** [comment]
From the discussion two weeks ago, @tobyhodges had an idea on how to formulate sth. that sounds convincing of how to retain RSEs in academia and what would be good arguments to actually do this. I feel this should be part of the call for action paper and the organization paper.
related issues:
https://github.com/CaptainSifff/paper_teaching-learning-RSE/pull/78 https://github.com/CaptainSifff/paper_teaching-learning-RSE/pull/84
pinging some contributors:
@mschwarzmeier @Bisaloo @jngrad @jlinx

**Comment by CaptainSifff** [comment]
and the comment by @Bisaloo

There are useful elements on this topic here: https://zenodo.org/record/8264153
